Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement_test.py

```diff
@@ -255,3 +249,41 @@ def test_get_asset_internet_exposure_command(requests_mock):
     assert response.outputs == EXTERNAL_EXPOSURE_RESULTS
     assert response.outputs_prefix == 'ASM.AssetInternetExposure'
     assert response.outputs_key_field == 'asm_ids'
+
+
+def test_list_remediation_rule_command(requests_mock):
+    """Tests list_remediation_rule_command function.
+
+        Given:
+            - requests_mock instance to generate the appropriate list_remediation_rule_command( API response,
+              loaded from a local JSON file.
+        When:
+            - Running the 'list_remediation_rule_command'.
+        Then:
+            - Checks the output of the command function with the expected output.
+    """
+    from CortexAttackSurfaceManagement import Client, list_remediation_rule_command
+
+    from test_data.raw_response import REMEDIATION_RULES_RESPONSE
+    from test_data.expected_results import REMEDIATION_RULES_RESULTS
+    requests_mock.post('https://test.com/api/webapp/public_api/v1/xpanse_remediation_rules/rules/',
+                       json=REMEDIATION_RULES_RESPONSE)
+
+    client = Client(
+        base_url='https://test.com/api/webapp/public_api/v1',
+        verify=True,
+        headers={
+            "HOST": "test.com",
+            "Authorizatio": "THISISAFAKEKEY",
+            "Content-Type": "application/json"
+        },
+        proxy=False)
+    args = {
+        'asm_rule_id': 'RdpServer'
+    }
+
+    response = list_remediation_rule_command(client, args)
+
+    assert response.outputs == REMEDIATION_RULES_RESULTS
```

Comment:
Not necessary to do for this, but if you ever need to increase code coverage you can also import `CommandResults` from CommonServerPython. You can create a CommandResults object in the test and set it to a variable and check you function output to it. 

Then you could assert it like this:
```python
assert command_results_from_your_function.to_context() == expected_command_results.to_context()
```

Reply:
thanks for the input, i can keep this in mind for later